Tasks/AzureWebAppV1/deploymentProvider/AzureRmWebAppDeploymentProvider.ts

```diff
@@ -39,9 +39,7 @@ export class AzureRmWebAppDeploymentProvider implements IWebAppDeploymentProvide
             this.kuduService = new Kudu(this.publishProfileScmCredentials.scmUri, this.publishProfileScmCredentials.username, this.publishProfileScmCredentials.password);
             let resourceId = publishProfileEndpoint.resourceId;
             let resourceIdSplit = resourceId.split("/");
-            if (resourceIdSplit.length === 11) {
-                this.slotName = resourceIdSplit[10];
```

Comment:
can we not just use `this.taskParams.SlotName` for slotName

Reply:
In the case of publish profile, the input fields related to webapp are redundant. The artifact will be deployed only at the scm url present in the publish profile, irrespective of what the user provides value into input fields.

Reply:
Lets take this out of this PR